### ISSUE TITLE
Hide Maven view by default

### ISSUE THREAD

**Issue by chambery** [Microsoft/vscode-maven#51]
Maven Projects view should hide if the project isn't Maven-enabled.  I added `when` to the `package.json` :
```
		"views": {
			"explorer": [
				{
					"id": "mavenProjects",
					"name": "Maven Projects",
					"when": "workspaceContains:pom.xml"
				}
```
which succeeds in hiding when I don't have a Maven project, but doesn't show in the view contributions list when I load my Java project containing a `pom.xml`.

**Comment by Eskibear** [comment]
can be done by Context value, value is set by `command.setContext` API

**Comment by Eskibear** [comment]
fixed in #80
